perfkitbenchmarker/linux_packages/maven.py

```diff
@@ -14,12 +14,17 @@
 """Module containing maven installation functions."""
 
 import posixpath
+import os
```

Comment:
Reorder the import:

import os
import posixpath

from perfkitbenchmarker import data
from perfkitbenchmarker import flags
from perfkitbenchmarker.linux_packages import INSTALL_DIR
from six.moves.urllib.parse import urlparse